File: AnalisisUberPOO/Javascript/Car.js
/*function Car(licence,driver,passenger){
    this.id;
    this.licence = licence;
    this.driver = driver;
    this.passenger = passenger;
}

Car.prototype.printDataCar = function(){
    console.log(this.driver);
    console.log(this.driver.name);
    console.log(this.driver.document);
}*/

class Car{

    constructor(license,driver,passenger){
        this.id;
        this.license = license;
        this.driver = driver;
        this.passenger = passenger;
    }

    printDataCar() {
        console.log(this.driver);
        console.log(this.driver.name);
        console.log(this.driver.document);
    }

}

File: AnalisisUberPOO/Java/Car.java
package Java;

public class Car {
    Integer id;
    String licence;
    Account driver;
    Integer passagenger;

    public Car(String licence, Account driver){
        this.licence = licence;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("license: "+licence+" Driver: "+driver.name);
    }

}
